spec/support/aruba_support.rb

```diff
@@ -10,7 +10,7 @@ module ArubaLoader
   let(:stderr) { StringIO.new }
   let(:stdout) { StringIO.new }
 
-  attr_reader :last_cmd_stdout, :last_cmd_stderr
+  attr_reader :last_cmd_stdout, :last_cmd_stderr, :last_cmd_exitstatus
```

Comment:
I'd prefer `last_cmd_exit_status` to `last_cmd_exitstatus`.  "Exit" and "status" are separate words.